libs/GameSharedResources/GameClock.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** GameClock
+*/
+
+#include "GameClock.hpp"
+
+using namespace std::chrono;
+
+namespace ecs
```

Comment:
Why this sudden change of using 'namespace ecs {}" instead of  "using namespace ecs" ?

Reply:
Sorry my bad